docs/src/color-system.js

```diff
@@ -3,9 +3,55 @@ import PropTypes from 'prop-types'
 import chroma from 'chroma-js'
 import styled from 'styled-components'
 import {Box, Text} from '@primer/components'
-import {colors, getPaletteByName} from './color-variables'
+import {colors, colorModes, getPaletteByName} from './color-variables'
 import Table from './table'
 
+function capitalize(word) {
+  return word[0].toUpperCase() + word.substr(1)
+}
+
+export function ColorModeTable({baseColor, values, ...rest}) {
+  const fgColor = overlayColor(baseColor)
+  const colorProps = {bg: baseColor, color: fgColor}
+
+  values.forEach(({variable, values, slug}) => {
+    if (!values.light || !values.dark) {
+      console.log(variable, values)
+    }
+  })
+
+  return (
+    <Table {...rest}>
+      <thead>
+        <tr>
+          <PaletteCell as="th" {...colorProps}>
+            Variable
+          </PaletteCell>
+          {colorModes.map(mode => (
+            <PaletteCell as="th" {...colorProps} key={mode}>
+              {capitalize(mode)} Mode
```

Comment:
Note to self: move `capitalize` to CSS

Reply:
Would this make `state-hoverPrimaryBg` into `state-hover-primary-bg`? That would be nice for copy&pasta.

Reply:
I used the [kebabCase](https://lodash.com/docs/4.17.15#kebabCase) function from lodash to convert `hoverPrimerBg` to `hover-primer-bg` in https://primer.style/primitives